src/select-list-view.ts

```diff
@@ -1,19 +1,42 @@
-const {Disposable, CompositeDisposable, TextEditor} = require('atom')
-const etch = require('etch')
+import { Disposable, CompositeDisposable, TextEditor, CommandEvent } from 'atom'
+// @ts-ignore Merge https://github.com/atom/etch/pull/90
+import etch from 'etch'
 const $ = etch.dom
-const fuzzaldrin = require('fuzzaldrin')
+import fuzzaldrin from 'fuzzaldrin'
+
+export type EtchElement = HTMLElement
+type EtchScheduler = any
+
+import { SelectListProperties } from './select-list-properties'
 
 module.exports = class SelectListView {
-  static setScheduler (scheduler) {
+  /** When creating a new instance of a select list, or when calling `update` on an existing one,
+  you can supply an object with the typeof SelectListProperties */
+  props: SelectListProperties
+
+  /** an array containing the objects you want to show in the select list. */
```

Comment:
```suggestion
  /** An array containing the filtered and ordered items to be shown in the select list. */
```

Would also remove the newline separating this one from all the other private variables.